lesson2/MainActivity.java/lib/src/main/java/com/example/Cat.java

```diff
@@ -8,13 +8,12 @@
 public class Cat extends Animal{
 
     public Cat(String name, String color) {
-        super(4,new Random().nextInt(25),"Cat", name, color);
+        super(4, new Random().nextInt(25), "Cat", name, color);
```

Comment:
Generally, adding a space before each new thing after a comma is good java style.